CRM/Import/Parser.php

```diff
@@ -1636,7 +1636,7 @@ protected function getTransformedFieldValue(string $fieldName, $importedValue) {
       if ($fieldMetadata['name'] === 'campaign_id') {
         if (!isset(Civi::$statics[__CLASS__][$fieldName][$importedValue])) {
           $campaign = Campaign::get()->addClause('OR', ['title', '=', $importedValue], ['name', '=', $importedValue], ['id', '=', $importedValue])->addSelect('id')->execute()->first();
-          Civi::$statics[__CLASS__][$fieldName][$importedValue] = $campaign['id'] ?? FALSE;
```

Comment:
So the reason this would be FALSE rather than NULL is because if it is NULL - then look ups would keep happening because the `!isset()` above would always be TRUE